apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -142,6 +140,7 @@ private void RunRetryTest(TestResponse response, TestContext context, string gro
         var result = ExecuteRpc(rpc, context, specifyPreconditions);
 
         // For list methods, we need to fetch a page in order to actually perform requests.
+        // Dont need to do this for notifications as those are getting consumed immidiately
         if (rpc.Contains("list") && !rpc.Contains("notification"))
         {
             ConsumeListOutput((dynamic) result);
```

Comment:
Even with your comment replying to my comment, I still don't understand why we went from 100 to 1000. Why is the fact that we are not deleting resources significant here?